src/ui/Pagination.scss

```diff
@@ -0,0 +1,76 @@
+$widget-pagination-color-active: #0595DB;
+$widget-pagination-color-disabled: #999;
```

Comment:
Check with Remco if these colors should come from atlas UI theming

Reply:
Please share his details. I also fee it should come from  the atlas ui